lib/file-system.js

```diff
@@ -7,6 +7,15 @@ exports.exists = function(path) {
   return new Promise((resolve, reject) => fs.exists(path, resolve));
 };
 
+exports.stat = function(path) {
```

Comment:
I am considering to drop a deprecation warning in here and remove completely later (as some new functions on the master branch that are not present here still use it).